wallet/api/session/sessions.go

```diff
@@ -13,6 +14,7 @@ var (
 	ErrNoWalletConnected                       = errors.New("no wallet connected")
 	ErrCannotEndLongLivingSessions             = errors.New("sessions attached to long-living tokens cannot be ended")
 	ErrGeneratedTokenCollidesWithExistingToken = errors.New("the generated token collides with an existing token")
+	ErrAPITokenExpired                         = errors.New("api token expired")
```

Comment:
It is user facing, we need something very user friendly

```suggestion
	ErrAPITokenExpired                         = errors.New("the token has expired")
```